test/functional/page_objects/common_page.ts

```diff
@@ -281,6 +281,13 @@ export class CommonPageObject extends FtrService {
           await this.browser.setLocalStorageItem('data.autocompleteFtuePopover', 'true');
           await this.browser.setLocalStorageItem('data.newDataViewMenu', 'true');
         }
+        if (appName === 'dashboard') {
+          const disableTour = JSON.stringify({
+            isTourActive: false,
+          });
+          await this.browser.setLocalStorageItem('dashboard.edit.tourState', disableTour);
+          await this.browser.setLocalStorageItem('dashboard.view.tourState', disableTour);
+        }
```

Comment:
Unfortunately I don't have access to the `dashboard_page` from inside `common_page`, so I had to duplicate the `forceSkipTour` code here... Not sure if there is a better way to do this? 🤷